src/components/Thread.js

```diff
@@ -130,7 +143,7 @@ class ThreadInner extends React.PureComponent {
       <React.Fragment>
         <MessageList
           messages={this.props.threadMessages}
-          headerComponent={headerComponent}
+          HeaderComponent={headerComponent}
```

Comment:
Is this new or a breaking change?

Reply:
Not breaking. I have added support for both `headerComponent` as well as `HeaderComponent`. Since its a component, its better to have prop name starting with capital letter.

Reply:
`headerComponent ` is marked as deprecated now.

Reply:
perfect